styles.css

```diff
@@ -1,52 +1,96 @@
 h1 {
-    font-family: 'Luminari';
-    color: blue;
-    font-size: 30px;
+  font-family: "Luminari";
+  color: blue;
+  font-size: 30px;
 }
 
 ul {
-    list-style-type: none;
-}
-
-ol {
-    font-weight: bold;
+  list-style-type: none;
 }
 
 li {
-    text-align: center;
-    border: 1px solid red;
-    border-radius: 10px;
-    color: blue;
-    background-color: lightpink;
-    padding-left: 30px;
+  text-align: center;
+  border: 1px solid red;
+  border-radius: 10px;
+  color: blue;
+  background-color: lightpink;
+  padding-left: 30px;
 }
 
 img {
-    width: auto;
-    height: 200px;
-    display: block;
-    margin-left: auto;
-    margin-right: auto;
+  width: auto;
+  height: 200px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
 }
 
 body {
-    font-family: sans-serif;
+  font-family: sans-serif;
 }
 
 h3 {
-    text-decoration-line: underline;
+  text-decoration-line: underline;
 }
 
 html {
-    background-color: rgb(159,193,201);
-    background: radial-gradient(circle, rgba(159,193,201,1) 9%, rgba(224,209,221,1) 52%, rgba(0,212,255,1) 100%);
+  background-color: rgb(159, 193, 201);
+  background: radial-gradient(
+    circle,
+    rgba(159, 193, 201, 1) 9%,
+    rgba(224, 209, 221, 1) 52%,
+    rgba(0, 212, 255, 1) 100%
+  );
 }
 
 p {
-    padding-top: 20px;
-    padding-bottom: 20px;
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+h2,
+h3 {
+  font-size: 20px;
+}
+
+.important {
+  font-weight: bold;
+  text-decoration: underline;
+}
+
+#skills-header {
+  color: rebeccapurple;
+}
+
+a[href$=".com"]:visited {
+  color: green;
+}
+
+.links-list {
+  list-style: outside url("https://github.githubassets.com/images/icons/emoji/unicode/1f310.png")
+    disc;
 }
 
-h2, h3 {
-    font-size: 20px;
-}
\ No newline at end of file
+ol li:last-of-type {
+  font-weight: bold;
+}
+
+a[href$=".com"]:focus {
+  color: red;
+}
+
+a:not([href$=".com"]):focus {
+  color: orange;
+}
+
+body {
+  text-align: center;
+}
+
+#language-list {
+  font-style: italic;
+}
+
+p:nth-child(3) {
+  background-color: rgba(0, 0, 0, 0.5);
+}
```

Comment:
Is this solution code? Make sure solution code is not in this particular PR or else we'll end up merging the solution into the lab and give away the answers.

Reply:
Yeah, I'd pushed it to the wrong PR 😄 . The other one is ready for review.